firmware/usbarmory/README.md

```diff
@@ -373,11 +373,18 @@ configuration.
 
 Then run `make`, as shown below, to build U-Boot:
 
+Linux
 ``` console
 $ # NOTE this depends on arm-none-eabi-gcc and other C build tools
 $ ARCH=arm CROSS_COMPILE=arm-none-eabi- make
 ```
 
+On Mac, you need to work around two things.
```

Comment:
Should we also consider writing a small Dockerfile for building U-Boot as well?

Reply:
Another option is extracting the u-boot image from a [pre-compiled Debian image](https://github.com/f-secure-foundry/usbarmory-debian-base_image/releases). I'm checking if that works right now; if it works I'll send a PR to this branch to document that as a simpler alternative.